conanfile.py

```diff
@@ -70,6 +65,10 @@ def configure_cmake(self):
             cmake.definitions["NOVELRT_BUILD_SAMPLES"] = "On"
         else:
             cmake.definitions["NOVELRT_BUILD_SAMPLES"] = "Off"
+        if(self.options.buildinterop):
+            cmake.definitions["NOVELRT_BUILD_INTEROP"] = "On"
+        else:
+            cmake.definitions["NOVELRT_BUILD_INTEROP"] = "Off"
```

Comment:
It was missing from a previous commit - I _could_ separate it but adding 4 lines in a separate PR makes no sense to me.